### ISSUE TITLE
GUI: DiskSwapper panel not displayed correctly [ + wiki does not detail hotkeys ] 

### ISSUE THREAD

**Issue by giantclambake** [BlitterStudio/amiberry#1118]
In the wiki we see this;

![wiki disk swapper image](https://i.imgur.com/anWM2B3.png)

With the current git-master build, I see this;

![current GUI image](https://i.imgur.com/OTEQQBQ.png)

I take it this is some WIP, but clearly this panel needs a scrollbar so user can navigate to/use the 'Remove All' button.

**Note:** another useful button here would be 'Add Multiple' ~ the ability to add multiple disk image files in one go.

In the helptext for the diskswapper panel, we can read;

![current helptext](https://i.imgur.com/LRGw6cz.png)

However in the wiki, no mention of the disk-swapper hotkey functionality is clearly documented ~ it merely intimates hotkeys can be used but not which hotkeys..ie; "_You can use this panel to insert multiple floppy disk images (e.g. ADF files) to the multiple slots, and quickly swap between them with hotkey combinations._"

I think this is probably an oversight, in as much as (builtin) program documentation should match wiki content (or vice-versa =)

TIA

**Comment by midwan** [comment]
The wiki has been updated with the shortcuts (which are coming from WinUAE): https://github.com/BlitterStudio/amiberry/wiki/GUI:-DiskSwapper

I'll update the screenshot later

**Comment by midwan** [comment]
Actually, there was a bug introduced here with one of the merges from WinUAE.
DiskSwapper should have only 10 slots as shown in the wiki screenshots, not 20. It was increased to 20 by mistake, but I've fixed that in the `preview` branch for now (will cherry pick this back to `master` later).

The description has been updated to better indicate the shortcuts available also. Only the first 4 drives can be used with Insert/Eject disk shortcuts, and those are END + F1-F4 (Insert), or SHIFT + END + F1-F4 (Eject).

I'll have to check if that description about using the numbers actually does anything, I don't recall if I added more shortcuts for swapping disks into drive DF0: for example. :)
